base/applications/network/tracert/tracert.cpp

```diff
@@ -340,7 +356,7 @@ DecodeResponse(
         }
         if (Status == IP_TTL_EXPIRED_TRANSIT || Status == IP_SUCCESS)
         {
-            PrintHopInfo(AddressInfo);
+            PrintHopInfo(AddressInfo, true);
```

Comment:
this feels a bit odd. Could you not have left the /r/n in the resources and just added a OutputText(IDS_LINEBREAK); here instead?

Reply:
Of course, fixed.